lib/rex/proto/ldap.rb

```diff
@@ -21,6 +30,13 @@ def read(length = nil, opts = {})
     end
   end
 
+  # Initialize the LDAP connection using Rex::Socket::TCP,
+  #   and optionally set up encryption on the connection if configured.
```

Comment:
```suggestion
  # Initialize the LDAP connection using Rex::Socket::TCP,
  # and optionally set up encryption on the connection if configured.
```

![image](https://user-images.githubusercontent.com/58950994/210890730-8d6f0380-4d8c-45b0-b310-7745879b361f.png)

Reply:
Darn it, thought I resolved all of these, thanks for pointing this out!

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/17427/commits/e03fd42a299a18ec7ed64ad908ddb9af9154ae39